src/views/CustomToken.vue

```diff
@@ -172,6 +178,10 @@ export default {
       }
       return null
     },
+    tokenError() {
+      // TODO :: Need to add translations
+      return 'Already exists. Go to Token Assets.'
```

Comment:
@mpwanyi256 you can add this to the English translation file and get it from there.

Reply:
@Jennievon Yes I needed to do that. I just wans't sure how. But I have just updated the english translation file.

Thanks